### DISCUSSION TITLE
Remove limitation for min/max year (1970-2038)

### DISCUSSION THREAD

**Discussion by adevade** [spatie/holidays#192] [Ideas]
Since the package switched from [`easter_date`](https://www.php.net/manual/en/function.easter-date.php) to [`easter_days`](https://www.php.net/manual/en/function.easter-days.php) to calculate easter, it should be possible to remove the min and max year limitation, right?

> **easter_days()**
> This function can be used instead of easter_date() to calculate Easter for years which fall outside the range of Unix timestamps (i.e. before 1970 or after 2037).

---

https://github.com/search?q=repo%3Aspatie%2Fholidays%20%22easter_date%22&type=code

Currently only [Kenya has to be refactored](https://github.com/spatie/holidays/blob/6e152ff4c77e4584a020259e0299af35e0f73756/src/Countries/Kenya.php#L31) to use the new easter method.

**Comment by inDeev** [comment]
Who would be able to search all historical changes in holidays from begining of ages? It was really frustrating with limitation to 1970 as there are so many changes in Czechia for example

**Reply by arnebr** [reply]
Maybe we should enable a country override, then each country could limit for themselves
